core-processor/src/configs.rs

```diff
@@ -24,7 +24,8 @@ use gear_core::{
     pages::WasmPagesAmount,
 };
 use gear_lazy_pages_common::LazyPagesCosts;
-use gear_wasm_instrument::syscalls::SyscallName;
+
```

Comment:
Why?))

Reply:
configs of core-processor takes this enum as args, so its useful to have public reimport here for ease of usage